src/blockchain/utils.ts

```diff
@@ -407,12 +407,20 @@ export const getTokenSymbol = (symbol: ITokenSymbols) => {
   }
 }
 
-export const getLPTokenSymbol = (symbol: ILPTokenSymbols): IU3LPToken => {
+export const getLPTokenSymbol = (
+  symbol: ILPTokenSymbols,
+  provider: BaseProvider
```

Comment:
By default, assume that the `chainId` is `Mainnet` and pass the `chainId` from wherever you're calling this method, instead of passing the `provider` itself which will need this function to be `async` 
```suggestion
  chainId = 1
```